libvast/src/system/partition.cpp

```diff
@@ -80,55 +80,55 @@ behavior collector(stateful_actor<collector_state>* self, predicate pred,
   };
 }
 
-struct bitmap_evaluator {
-  bitmap_evaluator(const std::unordered_map<predicate, bitmap>& bitmaps)
-    : bitmaps{bitmaps} {
+struct ids_evaluator {
+  ids_evaluator(const std::unordered_map<predicate, ids>& xs)
+    : xs{xs} {
     // nop
   }
 
-  bitmap operator()(none) const {
+  ids operator()(none) const {
     return {};
   }
 
-  bitmap operator()(conjunction const& c) const {
-    auto bm = visit(*this, c[0]);
-    if (bm.empty() || all<0>(bm))
+  ids operator()(conjunction const& c) const {
+    auto ys = visit(*this, c[0]);
```

Comment:
This is more of a reflective comment (and your input, @Neverlord, is appreciated). I like `xs`, `ys`, and `zs` as generic set types. But I wonder whether we should establish a specific name for a value that we're returning, e.g., `result`. That makes it clearer that we're working on an instance that will leave the function.

Reply:
I think `xs`, `ys`, and `zs` for input and temporary variables and `result` for whatever is returned is a good guideline.

Reply:
Alrighty, I'll update our style guide.